config/software/ruby.rb

```diff
@@ -50,6 +50,16 @@
 dependency "libiconv"
 
 # The SHA256 checksums below are for the *.tar.gz packages from https://www.ruby-lang.org/en/downloads/releases.
+version("3.1.3")      { source sha256: "5ea498a35f4cd15875200a52dde42b6eb179e1264e17d78732c3a57cd1c6ab9e" }
+version("3.1.2")      { source sha256: "61843112389f02b735428b53bb64cf988ad9fb81858b8248e22e57336f24a83e" }
```

Comment:
Do we need these intermediate versions that are not used?

Reply:
They don't hurt, and make this file closer to the one in omnibus-software…